src/sonic-frr-mgmt-framework/templates/bgpd/bgpd.spine_chassis_frontend_router.conf.j2

```diff
@@ -0,0 +1,63 @@
+!
+{# VNET BGP Instance #}
+! Vnet BGP instance
+{% set interfaces_in_vnets = [] %}
+{% block vnet_bgp_instance %}
+{%  for vnet_name, vnet_metadata in VNET.items() %}
+router bgp {{ DEVICE_METADATA['localhost']['bgp_asn'] }} vrf {{ vnet_name }}
+  no bgp default ipv4-unicast
```

Comment:
We dont need this file, please remove the same.

Reply:
Addressed